docs/api.md

```diff
@@ -550,6 +550,18 @@ const logger = pino(pinoOptions)
 * See [`message` log method parameter](#message)
 * See [`logMethod` hook](#logmethod)
 
+<a id="errors"></a>
```

Comment:
This identifier is very generic and will likely cause conflicts at some point. I suggest using "error-serialization" as the name.